app/controllers/miq_policy_controller.rb

```diff
@@ -943,7 +943,9 @@ def folder_get_info(folder_node)
     @sb[:folder] = nodeid.nil? ? nodetype.split("-").last : nodeid
     if x_active_tree == :policy_tree
       if nodeid.nil? && ["compliance", "control"].include?(nodetype.split('-').last)
-        @folders = ["Host #{nodetype.split('-').last.titleize}", "Vm #{nodetype.split('-').last.titleize}"]
+        @folders = ["Host #{nodetype.split('-').last.titleize}", "Vm #{nodetype.split('-').last.titleize}",
+                    "Container Image #{nodetype.split('-').last.titleize}"]
```

Comment:
The `@folders` here and below are repeated, have different meanings but are both tied to the classes: Host, Vm, ContainerImage because you had to make a very similar change to both.  Maybe these can be refactored so the controller is not so knowledgeable of what folders MiqPolicy supports.